tests/test_images.py

```diff
@@ -0,0 +1,61 @@
+import matplotlib.pyplot as plt
+import numpy as np
+import pytest
+
+from geograypher.utils.image import perspective_from_equirectangular
+
+
+@pytest.mark.parametrize(
+    "expected,yaw_deg,pitch_deg",
+    (
+        [[(45, 270), (90, 315), (135, 270), (90, 225)], 90, 0],
+        [[(45, 0), (90, 45), (135, 0), (90, 315)], 180, 0],
+    ),
+)
+def test_equi_to_perspective(expected, yaw_deg, pitch_deg):
+    output_size = (91, 91)
+    fov_deg = 90
+    oversample_factor = 1
+    warp_order = 0
+
+    # Create a test equirectangular image where the pixel values are the (i,j) coordinates
+    i_samples = np.arange(180)
+    j_samples = np.arange(360)
```

Comment:
This works, but maybe an arange from -180 to 180 instead of 0 to 360 would convey that the center of the image is the default / zero camera location? I puzzled on that for a second until I realized the `ij` matrix was arbitrary